CMakeLists.txt

```diff
@@ -1263,6 +1264,25 @@ ELSE(LIBXML2_FOUND)
     SET(HAVE_LIBEXPAT 1)
     LA_CHECK_INCLUDE_FILE("expat.h" HAVE_EXPAT_H)
     CMAKE_POP_CHECK_STATE()	# Restore the state of the variables
+  ELSE(EXPAT_FOUND)
+    IF(WIN32 AND ENABLE_WIN32_XMLLITE)
+      # Check linkage as well; versions of mingw-w64 before 5ac1a2c
```

Comment:
We could get around this by doing a runtime check - however, we would need to check during format initialization.

If you're alright with a `dlopen`/`dlsym` during format bidding, I would be too.

The upside is that libarchive can be built with (older) mingw-w64 and still retain xmllite support; the downside is that every time somebody calls `archive_read_support_format_xar` it's going to try to load up xmllite.dll.